resources/config.test.edn

```diff
@@ -8,7 +8,6 @@
                                    :port    [8125 :int]
                                    :enabled [false :bool]}
             :metrics              {:constructor "ziggurat.dropwizard-metrics-wrapper/->DropwizardMetrics"}
-            :alpha-features       {:protobuf-middleware {:enabled false}}
```

Comment:
can you just keep the :alpha-features key, this will remind people to add alpha specific config in this key

Reply:
done